app/src/components/blogs/data/Kashi-Vishwanath-Darshan.json

```diff
@@ -1,6 +1,6 @@
 {
     "title": "Kashi-Vishwanath-Darshan",
-    "image": "../images/blogKashivishwanath.png",
+    "image": "../images/blogKashiVishwanath.jpg",
```

Comment:
Why this change?

Reply:
replaced old kashiVishwanath image with new chat gpt one . renamed it same as old one just the format was different

Reply:
Okay

Reply:
Please add Screenshot